src/AcceptanceTests/ConfigureEndpointAzureStorageQueueTransport.cs

```diff
@@ -1,42 +0,0 @@
-using System.Threading.Tasks;
-using NServiceBus;
-using NServiceBus.AcceptanceTesting.Support;
-
-public class ConfigureEndpointAzureStorageQueueTransport : IConfigureEndpointTestExecution
```

Comment:
By removing ASQ here potential regressions in the hybrid mode would not be caught, see https://github.com/Particular/NServiceBus.AzureStorageQueues/issues/301

I'm not against these changes I'm just bringing this up for future reference.

@bording, even more, reason to push for the 7.1 changes you are advocating for?

Reply:
>By removing ASQ here potential regressions in the hybrid mode would not be caught, see 

Since the transport "owns" the hybrid mode would it make more sense for it to pull in ASP and run those tests there?

Reply:
we could have two acceptance tests projects similar to nhibernate or sqlp to run with different configurations?

Reply:
> Since the transport "owns" the hybrid mode would it make more sense for it to pull in ASP and run those tests there?

I agree ASQ should own those tests, I am skeptical the tests need to be against ASP.